cmd/migrate/postgres.go

```
@@ -182,3 +189,232 @@ func runPostgresMigration(dryRun bool, timeout int, allowIndexChanges bool) erro
+	action := "up"
+	if statusOnly {
+		action = "status"
+	}
+
+	fmt.Fprintf(os.Stderr, "dbmate %s against %s:%d/%s (dir %s)\n",
+		action, cfg.Postgres.Host, cfg.Postgres.Port, cfg.Postgres.DBName, dir)
+
+	// --no-dump-schema: dbmate writes schema.sql after a successful run by
+	// default, which needs pg_dump on PATH and a writable tree. Neither holds in
+	// the deployment image, and the dump is a local development convenience.
+	c := exec.Command(bin, "--migrations-dir", dir, "--no-dump-schema", action)
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)